python/conda/meta.yaml

```diff
@@ -20,12 +19,15 @@ requirements:
   build:
     - python
     - cython >=0.13
+    - nomkl # [not win]
```

Comment:
Update: Looks good. Just built both `libastra` and `astra-toolbox` on my laptop, moved the packages to my desktop machine, installed both in a fresh conda environment, once with mkl, once with nomkl. Both versions work.